custom_iterm_script_iterm_3.1.1.applescript

```diff
@@ -0,0 +1,45 @@
+-- This is v0.6 of the custom script for AlfredApp for iTerm 2.9+
```

Comment:
You should probably change the header to update the version and include a URL so when it's copied into iTerm people will know where it came from.

Reply:
I'm not sure how to do it.
Will you please update it as you have declared?

Reply:
Change this text to include the URL to this file.